allennlp/interpret/attackers/attacker.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+from allennlp.common import Registrable
+from allennlp.predictors import Predictor
+from allennlp.common.util import JsonDict
+
+class Attacker(Registrable):
+    """
+    an ``Attacker`` will modify an input (e.g., add or delete tokens)
+    to try to change an AllenNLP Predictor's output in a desired
+    manner (e.g., make it incorrect).
+    """
+    def __init__(self, predictor: Predictor) -> None:
+        self.predictor = predictor
+
+    def attack_from_json(self,
+                         inputs: JsonDict,
+                         input_field_to_attack: str,
+                         grad_input_field: str,
+                         ignore_tokens: List[str]) -> JsonDict: # pylint: disable=dangerous-default-value
```

Comment:
You don't need this disable anymore.

Reply:
for some reason, pylint has this dangerous-default-value on the inputs field everywhere.

Reply:
Yes, because you're using a dangerous default value in those places.  You need to fix that and use `None` as the default value, and remove the warning.  There's no default value on this method, though, so it shouldn't be appearing here.  If it is, it's a bug in pylint.